README.md

```diff
@@ -1,94 +1,46 @@
 # A Reference Process for OSS Projects on GitHub
 
-This is an attempt to document similarities in in a number of our OSS projects hosted on GitHub. 
+This is an attempt to document practices common to a number of OSS projects we have on GitHub. 
```

Comment:
@wkurniawan07 
If we say something like `prefix PR number with your matric number`, that's specific to NUS. None of the guidelines we have are specific to NUS. I tried to exclude any NUS specific things on purpose. If I missed any, do point out so that I can get rid of it. I think anyone running an OSS project on GitHub and has no preconceived idea of how to run it (e.g. if that's their first OSS project) can follow our process exactly as its prescribed and they'll do just fine. Any thoughts?

The point you raise, which is valid, is actually about the level of concreteness, not about the level of specificity. Our guidelines are very concrete (e.g. `copy paste issue title as PR title`) rather than abstract (e.g. `PR title should be clearly linked to issue title`). The concreteness, while being very much influenced by our personal preferences, also makes them immediately useful. It took a lot of trial-and-error for us to figure out those concrete details that works reasonably well. Some others (obviously not all) may be able to simply follow our final version of the process, instead of having to figure out themselves all those details. For example, PhD students who are developing OSS projects as part of their research can follow this process irrespective of whether they are in NUS or not. Those guys are unlikely to have lot of experience running OSS projects themselves, so they'll be starting with a clean slate. By simply copying our process, they can have a sizeable head start instead of starting from zero.

As I write this answer, I also realized one possible way we can narrow down the scope. Instead of claiming that it fits all OSS projects, we can say that it can help those who are new to running OSS projects, who can use this as a 'starter process' that they can adopt partially or fully at the start and then evolve as they see fit as they gain more experience and start developing their own ideas of how to do OSS things.